src/VkBootstrap.cpp

```diff
@@ -900,75 +906,15 @@ bool supports_features(VkPhysicalDeviceFeatures supported, VkPhysicalDeviceFeatu
-    if(requested.descriptorBindingSampledImageUpdateAfterBind && !supported.descriptorBindingSampledImageUpdateAfterBind) return false;
-    if(requested.descriptorBindingStorageImageUpdateAfterBind && !supported.descriptorBindingStorageImageUpdateAfterBind) return false;
-    if(requested.descriptorBindingStorageBufferUpdateAfterBind && !supported.descriptorBindingStorageBufferUpdateAfterBind) return false;
-    if(requested.descriptorBindingUniformTexelBufferUpdateAfterBind && !supported.descriptorBindingUniformTexelBufferUpdateAfterBind) return false;
-    if(requested.descriptorBindingStorageTexelBufferUpdateAfterBind && !supported.descriptorBindingStorageTexelBufferUpdateAfterBind) return false;
-    if(requested.descriptorBindingUpdateUnusedWhilePending && !supported.descriptorBindingUpdateUnusedWhilePending) return false;
-    if(requested.descriptorBindingPartiallyBound && !supported.descriptorBindingPartiallyBound) return false;
-    if(requested.descriptorBindingVariableDescriptorCount && !supported.descriptorBindingVariableDescriptorCount) return false;
-    if(requested.runtimeDescriptorArray && !supported.runtimeDescriptorArray) return false;
-    if(requested.samplerFilterMinmax && !supported.samplerFilterMinmax) return false;
-    if(requested.scalarBlockLayout && !supported.scalarBlockLayout) return false;
-    if(requested.imagelessFramebuffer && !supported.imagelessFramebuffer) return false;
-    if(requested.uniformBufferStandardLayout && !supported.uniformBufferStandardLayout) return false;
-    if(requested.shaderSubgroupExtendedTypes && !supported.shaderSubgroupExtendedTypes) return false;
-    if(requested.separateDepthStencilLayouts && !supported.separateDepthStencilLayouts) return false;
-    if(requested.hostQueryReset && !supported.hostQueryReset) return false;
-    if(requested.timelineSemaphore && !supported.timelineSemaphore) return false;
-    if(requested.bufferDeviceAddress && !supported.bufferDeviceAddress) return false;
-    if(requested.bufferDeviceAddressCaptureReplay && !supported.bufferDeviceAddressCaptureReplay) return false;
-    if(requested.bufferDeviceAddressMultiDevice && !supported.bufferDeviceAddressMultiDevice) return false;
-    if(requested.vulkanMemoryModel && !supported.vulkanMemoryModel) return false;
-    if(requested.vulkanMemoryModelDeviceScope && !supported.vulkanMemoryModelDeviceScope) return false;
-    if(requested.vulkanMemoryModelAvailabilityVisibilityChains && !supported.vulkanMemoryModelAvailabilityVisibilityChains) return false;
-    if(requested.shaderOutputViewportIndex && !supported.shaderOutputViewportIndex) return false;
-    if(requested.shaderOutputLayer && !supported.shaderOutputLayer) return false;
-    if(requested.subgroupBroadcastDynamicId && !supported.subgroupBroadcastDynamicId) return false;
+
+    for(auto i = 0; i < extension_requested.size(); ++i) {
+        //auto res = extension_requested[i].match(extension_supported[i]);
```

Comment:
Commented out code should be removed. (I forget about this all the time, easy to miss.)